### PR TITLE
[CN-693]Add new feature to README.md and CSV file

### PR DESCRIPTION
**Pull request by cheels**
## Description

Update README.md file and CSV files with new features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +32,17 @@ Hazelcast Platform Operator supports the features below:
...
 * Expose Hazelcast cluster to external
   clients ([Smart & Unisocket](https://docs.hazelcast.com/hazelcast/latest/clients/java#java-client-operation-modes))
 * Backup Hazelcast persistence data to cloud storage with the possibility of scheduling it and restoring the data accordingly
 * WAN Replication feature when you need to synchronize multiple Hazelcast clusters, which are connected by WANs
 * User Code Deployment feature, which allows you to deploy custom and domain classes from cloud storages to Hazelcast members
 * ExecutorService and EntryProcessor support
-* Support several data structures like Map, Topic, MultiMap, and ReplicatedMap, which can be created dynamically via specific Custom Resources
+* Support several data structures like Map, Topic, MultiMap, ReplicatedMap, Queue and Cache which can be created dynamically via specific Custom Resources
 * MapStore support for Map CR
+* Persistence support for the Cache CR
```
Comment (hasancelik) [inline README.md:45]: I think we can remove this line 
```suggestion

```

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: config/manifests/bases/hazelcast-platform-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -94,15 +94,17 @@ spec:
...
     * Expose Hazelcast cluster to external
       clients ([Smart & Unisocket](https://docs.hazelcast.com/hazelcast/latest/clients/java#java-client-operation-modes))
     * Backup Hazelcast persistence data to cloud storage with the possibility of scheduling it and restoring the data accordingly
     * WAN Replication feature when you need to synchronize multiple Hazelcast clusters, which are connected by WANs
     * User Code Deployment feature, which allows you to deploy custom and domain classes from cloud storages to Hazelcast members
     * ExecutorService and EntryProcessor support
-    * Support several data structures like Map, Topic, MultiMap, and ReplicatedMap, which can be created dynamically via specific Custom Resources
+    * Support several data structures like Map, Topic, MultiMap, ReplicatedMap, Queue and Cache which can be created dynamically via specific Custom Resources
     * MapStore support for Map CR
+    * Persistence support for the Cache CR
```
Comment (cheels) [inline config/manifests/bases/hazelcast-platform-operator.clusterserviceversion.yaml:107]:
```suggestion
```

Comment (cheels) [reply config/manifests/bases/hazelcast-platform-operator.clusterserviceversion.yaml:107]: @hasancelik Should I remove it from CSV file as well?

Comment (hasancelik) [reply config/manifests/bases/hazelcast-platform-operator.clusterserviceversion.yaml:107]: I have suggested this change for CSV file actually 🙂

**Pull request merged**